apps/feature/views/are-you-sure.html

```diff
@@ -0,0 +1,7 @@
+{{<partials-page}}
+  {{$back}}<a class="link-back" href="/feature/forms">{{#t}}buttons.back-to-reports{{/t}}</a><br>{{/back}}
+  {{$page-content}}
+    <p>Are you sure you wish to delete form with reference: {{reference}}?</p>
+    <button name="confirm" value="{{id}}" class="button">Delete this form</button>&nbsp; <button class="button" name="cancel">Cancel</button>
+  {{/page-content}}
+{{/partials-page}}
```

Comment:
You might need to add a title to the page - if nothing is appearing on the tab on chrome.

Reply:
added title to this, and added header to pages.json